### PR TITLE
Background

### PR DESCRIPTION
**Pull request by mariaalouisaa**
We had 9 colours on the figma but I trimmed it down to 8 to make it nicer on the eyes!

### CODE REVIEW

File: pages/learner-onboarding/background-selection.js [hunk 1]
```diff
@@ -1,12 +1,54 @@
 import LearnerOnboardingLayout from "../../components/LearnerOnboardingLayout";
+import Tts from "../../components/tts";
+import { colours } from "../../data/colours.js";
```
Comment (PJSalter) [inline pages/learner-onboarding/background-selection.js:3]: Good shout on the curly bracket for the name, this got us to fix our path later for the avatar ✨

File: pages/learner-onboarding/background-selection.js [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
+import styled from "styled-components";
+import { useState, useEffect } from "react";
 
 export default function LearnerBackgroudSelection() {
+  const [background, setBackground] = useState(null);
+
+  const updateColour = (e) => {
```
Comment (PJSalter) [inline pages/learner-onboarding/background-selection.js:10]: nice work on the updating colour 🎨

File: pages/learner-onboarding/background-selection.js [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
+    setBackground(e.target.value);
+  };
+
+  useEffect(() => {
+    document.body.style.backgroundColor = background;
```
Comment (PJSalter) [inline pages/learner-onboarding/background-selection.js:15]: Awesome ✨ 🖌️

File: pages/learner-onboarding/background-selection.js [hunk 4]
```diff
@@ -1,12 +1,54 @@
...
   return (
     <LearnerOnboardingLayout
       ttsTitle="Click the colours to change the background."
       stepNumber="4"
       nextStep="onboarding-done"
       previousStep="font-size-selection"
-    />
+    >
+      <Grid>
+        {colours.map((colour) => (
```
Comment (PJSalter) [inline pages/learner-onboarding/background-selection.js:26]: clear and understandable code 😸

File: pages/learner-onboarding/background-selection.js [hunk 5]
```diff
@@ -1,12 +1,54 @@
...
+  margin: 2rem 1rem 0 1rem;
+  cursor: pointer;
+  border: 2px grey solid;
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+  border-radius: 5px;
+`;
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (PJSalter) [inline pages/learner-onboarding/background-selection.js:51]: great use of grid column layout 🌱

File: pages/learner-onboarding/background-selection.js [hunk 6]
```diff
@@ -1,12 +1,54 @@
...
       nextStep="onboarding-done"
       previousStep="font-size-selection"
-    />
+    >
+      <Grid>
+        {colours.map((colour) => (
+          <ColourButton
+            style={{ backgroundColor: colour.hex }}
+            value={colour.hex}
+            onClick={updateColour}
```
Comment (PJSalter) [inline pages/learner-onboarding/background-selection.js:30]: the onclick of updating colour is great 👩‍🎨

**Review by PJSalter — APPROVED** [review]

**Review by joe-dev-public — APPROVED** [review]

**Pull request merged**